com.unity.render-pipelines.core/ShaderLibrary/Color.hlsl

```diff
@@ -254,7 +254,11 @@ real YCoCgCheckBoardEdgeFilter(real centerLum, real2 a0, real2 a1, real2 a2, rea
 }
 
 // Converts linear RGB to LMS
+#if defined(SHADER_API_SWITCH) // Must use full float precision on Switch to avoid artefact when using ACES tonemapping
 real3 LinearToLMS(real3 x)
```

Comment:
Did you mean to use float3 line 258 too?

Reply:
Yes you are right, thanks. Actually the function causing problem is LMSToLinear(), but I thought it was better to replicate the change to fp32 calculation on both functions.